.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+16
```

Comment:
any reason *not* to use lts?

Reply:
https://github.com/Uniswap/widgets/pull/32/files#r898373397

Reply:
Haha, I thought I remembered that! https://nodejs.org/en/about/releases/

at this point, it probably makes sense to bump it to 18 if we're going to make a change here